scripts/008_scored_survey.R

```diff
@@ -29,7 +31,7 @@ for(i in sep_cols) {
     w2 <- rbind(w2, w1)
 }
 
-#converting wide to long and addressing TODO: #8
+#converting wide to long
 w3 <- w2 %>%
 select(-c(affilitation, address, brief_summary)) %>%
```

Comment:
what happens when collaborator names are the same, we should use the affiliation

Reply:
noted, but will not be fixed